src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
таким чином ми тут не використовуємо поліморфізм, коректно буде так 
```suggestion
        Machine bulldozer = new Bulldozer();
```
аналогічно з іншими об'єктами

Reply:
Дякую